test/extractor/CardRuleExtractor.test.ts

```diff
@@ -43,9 +63,118 @@ describe('CardRuleExtractor', () => {
 
     it('should return a null when the element has no cardinality information', () => {
       const element = ProcessableElementDefinition.fromJSON(looseSD.differential.element[1]);
-      const cardRule = CardRuleExtractor.process(element);
+      const cardRule = CardRuleExtractor.process(element, looseSD, defs);
       expect(cardRule).toBeNull();
       expect(element.processedPaths).toEqual([]);
     });
+
+    it('should return a null when there is only a min, and there are card-constrained slices, and min matches the sum of mins of all the slices', () => {
+      const element = ProcessableElementDefinition.fromJSON(
+        looseSDWithSlices.differential.element[0]
+      );
+      const cardRule = CardRuleExtractor.process(element, looseSDWithSlices, defs);
+      expect(cardRule).toBeNull();
+      expect(element.processedPaths).toEqual(['min']);
+    });
+
+    it('should return a card rule with min when the min does not match the sum of mins of the slices', () => {
+      // Modify the fixture so the min doesn't equal the sum of the slices' mins
+      const clonedSD = cloneDeep(looseSDWithSlices);
+      clonedSD.differential.element[0].min = 4;
+      const element = ProcessableElementDefinition.fromJSON(clonedSD.differential.element[0]);
+      const cardRule = CardRuleExtractor.process(element, clonedSD, defs);
+      const expectedRule = new ExportableCardRule('component');
+      expectedRule.min = 4;
+      expect(cardRule).toEqual<ExportableCardRule>(expectedRule);
+      expect(element.processedPaths).toEqual(['min']);
+    });
+
+    it('should return a null when there is only a min, and there are card-constrained slices, and min matches the sum of mins of the all the slices (and the slicing is only in snapshot)', () => {
+      // Modify the fixture so the slicing is only in the snapshot (representing the case where it is inherited)
+      const clonedSD = cloneDeep(looseSDWithSlices);
+      delete clonedSD.differential.element[0].slicing;
+      const element = ProcessableElementDefinition.fromJSON(clonedSD.differential.element[0]);
+      const cardRule = CardRuleExtractor.process(element, clonedSD, defs);
+      expect(cardRule).toBeNull();
+      expect(element.processedPaths).toEqual(['min']);
+    });
+
+    it('should return a card rule with min even if it matches the sum of mins of the slices when there are no card-constrained slices in the snapshot', () => {
```

Comment:
If I understand this test case correctly, I think the description should be "when there are no card-constrained slices in the differential".

Reply:
You're right.  That was the result of changing what the test actually tested too many times!  Fixed in my last commit!